.github/workflows/pkg-generator-supabase.yml

```
@@ -0,0 +1,30 @@
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: Enable corepack (pnpm)
+        run: corepack enable
+      - name: Install deps
+        run: pnpm install --frozen-lockfile
+      - name: @wesley/generator-supabase tests
```

actionlint (1.7.8)

could not parse as YAML: found character that cannot start any token

(syntax-check)